src/box/lua/slab.c

```diff
@@ -254,6 +254,16 @@ lbox_runtime_info(struct lua_State *L)
 	return 1;
 }
 
+static int
+lbox_runtime_check_recover_complete(struct lua_State *L)
```

Comment:
According to our coding style that function must be named `is_recover_complete`

Reply:
ok